frontend/public/components/modals/cluster-channel-modal.tsx

```diff
@@ -22,34 +29,67 @@ const ClusterChannelModal = withHandlePromise((props: ClusterChannelModalProps)
     o[val] = val;
     return o;
   }, {});
+  const version = semver.parse(getLastCompletedUpdate(cv));
   const submit: React.FormEventHandler<HTMLFormElement> = (e) => {
     e.preventDefault();
     const patch = [{ op: 'add', path: '/spec/channel', value: channel }];
     return handlePromise(k8sPatch(ClusterVersionModel, cv, patch), close);
   };
+  const channelsExist = cv.status.desired.channels;
```

Comment:
I'd be careful and use optional chaining.

```suggestion
  const channelsExist = cv.status?.desired?.channels?.length;
```

Reply:
Thanks, @spadgett.  Updated.